### DISCUSSION TITLE
Moved Inscriptions by sending all BTC in wallet - now can't send inscriptions

### DISCUSSION THREAD

**Discussion by skyrover7** [ordinals/ord#1711] [Q&A] [answered]
I was doing some experimenting and tried the following steps. 

* created 3 inscriptions in wallet A
* using btc core wallet, sent all btc from wallet A to wallet B
* when i check Wallet B for inscriptions (`ord wallet inscriptions`), it shows all 3 inscriptions
* when i try to send an inscription from Wallet B to another wallet using `ord wallet send ADDRESS INSCRIPTION_ID`
* i get an error as below

`error: cannot send 3852aa011aafae4e84b52775e709a9fd62cf57769e9b57804b2b85c732a37f2a:0:143855 without also sending inscription 4b0a98b8c84112e6f63f7b6949f10d0eab65c5170cf034877d8f71a84efc2265i0 at 3852aa011aafae4e84b52775e709a9fd62cf57769e9b57804b2b85c732a37f2a:0:0`

**Comment by andrewhong5297** [comment]
guess wallet just doesn't support this for now, but I'm assuming in the future you can probably split the UTXO into three separate UTXOs again so that they can be sent separately. 

someone aggregated 166 inscriptions and haven't done anything with them yet lol https://twitter.com/andrewhong5297/status/1624085592131731459

have an open question here trying to get more clarity on rules for transferring sats/UTXOs without breaking inscriptions. https://github.com/casey/ord/discussions/1687

really a lack of guides/discussion on this that I can find at least. there's no contract, so standards will just have to be "accepted" but the longer it takes to create a "standard" the more random cases we're going to end up with from people just playing around haha.

**Reply by skyrover7** [reply]
Understood and thank you! Yea i assume in the future they'll be accessible again.

But for now, safe to say they are locked until further notice?

**Reply by andrewhong5297** [reply]
yeah unless you can program the transfer split yourself using bitcoin cli

**Reply by skyrover7** [reply]
hmm, is that a thing that is possible? I am happy to figure it out, but would need more details on what exactly i'm trying to do 😆 so i can search accordingly. let me know if you have any ideas or approaches and im happy to look into it.

Would really be great to have a solution to this, have spoken to multiple folks in the community that have this issue and are not able to access their inscriptions

**Comment by skyrover7** [comment]
@raphjaph any thoughts on this?

**Comment by raphjaph** [comment, marked as answer]
We do not recommend using `bitcoin-cli` to do any transactions involving inscriptions because it is not ordinal-aware. What happened in your case is that it consolidated all utxos into one. You are lucky that you didn't lose an inscription to fees! We perform a lot of checks to make sure that doesn't happen with the `ord wallet` command. We haven't implemented splitting up multiple inscriptions in a utxo yet but it is on the todo list. 

For your use case we are probably gonna implement `ord wallet sweep <ADDRESS>`(#1071 ) but we are a bit bottlenecked by dev capacity at the moment because this blew up faster than we expected.

**Reply by skyrover7** [reply]
@raphjaph you mean you weren't expecting to go from 5k to 100k in 7 days?! 😆 

thank you for the response, i'll keep tracking #1071

**Reply by nobullryder** [reply]
@raphjaph any updates on this?

**Comment by reinamora137** [comment]
Perhaps some more warnings are in order in the `ord --help` or otherwise..   I did an RBF transaction thinking 'oh this won't impact the inscriptions'. lol, was I wrong.

**Reply by skyrover7** [reply]
yea someone had warned me about that. i _kind of_ i had some success using CPFP on my first inscription where i did not set a fee rate (also something that would be useful to note in docs, because at default 1, that transaction will never clear). however in doing so i locked the inscriptions per the original discussion point.

I did have some success using a [third party accelerator](https://btcaccelerators.com/) though. Not sure if this is recommended or not but it did work

**Reply by reinamora137** [reply]
The external accelerators make sense as they wouldn't screw up the ord wallet. I attempted with btcaccelerators as you mentioned, but they wanted over $1000 USD for two transactions because they are double spend... Argh.

**Reply by skyrover7** [reply]
oh really? that doesn't sound right. i think i paid like .01 eth to speed up a send transaction. but perhaps because that was a really small transaction to start with. I assume you were trying to speed up an insscribe? or a send?
